actionbutton/actionbutton.js

```diff
@@ -32,32 +32,33 @@ async function applyActions() {
   }
 }
 
-function onRecord(row) {
+function onRecord(row, mapping) {
```

Comment:
Is it now possible to remove the added parameter, and simplify the `data.column = mapping ? mapping.Action : "ActionButton";` line, by using `mapColumnNames`?